connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -127,7 +127,7 @@ private enum SourceSink {
     @Before
     public void setup() {
         worker = PowerMock.createMock(Worker.class);
-        String[] methodNames = new String[]{"connectorTypeForClass"/*, "validateConnectorConfig"*/, "buildRestartPlan", "recordRestarting"};
+        String[] methodNames = new String[]{"connectorType"/*, "validateConnectorConfig"*/, "buildRestartPlan", "recordRestarting"};
```

Comment:
While we're in the neighborhood?
```suggestion
        String[] methodNames = new String[]{"connectorType", "buildRestartPlan", "recordRestarting"};
```